exercises/es/exc_03_14_01.py

```diff
@@ -1,12 +1,12 @@
 import json
 import spacy
 
-nlp = spacy.load("en_core_web_sm")
+nlp = spacy.load("es_core_news_sm")
 
-with open("exercises/en/tweets.json") as f:
+with open("exercises/es/tweets.json", encoding="utf8") as f:
     TEXTS = json.loads(f.read())
 
 # Procesa los textos e imprime los adjetivos en pantalla
 for text in TEXTS:
     doc = nlp(text)
-    print([token.text for token in doc if token.pos_ == "ADJ"])
+    print([token.text for token in doc if token.pos_ == "NOUN"])
```

Comment:
How accurate is the result here? 

```
['hamburguesa', 'libra']
['juguete']
[]
['horas', 'pedido']
['ganas']
```

Mostly asking because I was struggling with this in German and finding tweets that were somewhat interesting, without producing at least one super confusing tagger result.

Reply:
Hey, I just changed this to look for verbs, since the results make more sense like this. It worked before, but I had the sensation that it could be a bit confusing in certain cases, like the last one. I also struggled a lot to find the tweets and I had to modify them slightly for it to work properly, like removing the `@`, so it would recognize the names.

Reply:
Yeah, I had to do the same for the German, but I think it's worth it to not further confuse the user and distract from the _actual_ exercise and objective here.